cmd/kubeadm/app/cmd/alpha/kubelet.go

```diff
@@ -1,116 +0,0 @@
-/*
-Copyright 2018 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package alpha
-
-import (
-	"fmt"
-
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-	"k8s.io/apimachinery/pkg/util/version"
-	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
-	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
-	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
-	kubeletphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubelet"
-	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
-)
-
-var (
-	kubeletConfigEnableDynamicLongDesc = cmdutil.LongDesc(`
-		Enable or update dynamic kubelet configuration for a Node, against the kubelet-config-1.X ConfigMap in the cluster,
-		where X is the minor version of the desired kubelet version.
-
-		WARNING: This kubeadm feature is deprecated. Enable only if you know what you are doing, as it
-		may have surprising side-effects at this stage.
-
-		` + cmdutil.AlphaDisclaimer)
-
-	kubeletConfigEnableDynamicExample = cmdutil.Examples(fmt.Sprintf(`
-		# Enable dynamic kubelet configuration for a Node.
-		kubeadm alpha phase kubelet enable-dynamic-config --node-name node-1 --kubelet-version %s
-
-		WARNING: This kubeadm feature is deprecated. Enable only if you know what you are doing, as it
-		may have surprising side-effects at this stage.
-		`, constants.CurrentKubernetesVersion))
-)
-
-// newCmdKubeletUtility returns command for `kubeadm phase kubelet`
-func newCmdKubeletUtility() *cobra.Command {
```

Comment:
the first commit technically removes the whole `kubelet.go` file, because there are no-more commands under kubelet -> config after enable-dynamic is removed.